### PR TITLE
New event spec

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

File: actions/parse-push/src/run.ts [hunk 1]
```diff
@@ -160,19 +139,11 @@ async function getNewCases(commitSha: string): Promise<string[]> {
...
 async function parseCommit(commit: RemoteCommit): Promise<Commit | null> {
-    const slug = await getSlug(commit)
-    const pull_request_html_url = await getPullRequestHtmlUrl(commit.id)
-
-    if (!pull_request_html_url) {
-        return null
-    }
-
+    const slugs = await getSlugs(commit.modified)
     const new_cases = await getNewCases(commit.id)
```
Comment (SaschaMann) [inline actions/parse-push/src/run.ts]: Should this be fully async instead of waiting for them in order?

File: .gitattributes [hunk 2]
```diff
@@ -1,2 +1,2 @@
-dist/** -diff linguist-generated=true
+**/dist/** -diff linguist-generated=true
```
Comment (SleeplessByte) [inline .gitattributes:1]: This _should_ be the same.

- `/dist/**` would be dir paths inside the root dir dist only
- `dist/**` would be dir paths inside **any** dist
- `**/dist/**` would be "dir paths inside **any** dist that is inside a dir (which is included in the one above).

File: actions/parse-push/src/run.ts (hunk 1 above)
Comment (SleeplessByte) [reply actions/parse-push/src/run.ts]: It will be fast if we wait on both.

Comment (SleeplessByte) [inline actions/parse-push/src/run.ts]:
```suggestion
    const [slugs, new_cases] = await Promise.all([getSlugs(commit.modified), getNewCases(commit.id)])
```

Comment (SleeplessByte) [reply actions/parse-push/src/run.ts]: Not tested but should work

**Review by SleeplessByte — APPROVED** [review]

File: .gitattributes (hunk 2 above)
Comment (SaschaMann) [reply .gitattributes:1]: Before this change the index.js diff was displayed and not detected as generated 🤷

Comment (SleeplessByte) [reply .gitattributes:1]: Yeah, probably linguist that doesn't do this correctly :)

**Pull request merged**